src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$stopwatch-size: 80vmin;
+$center-size: $stopwatch-size / 20;
+$minute-hand-heigth: $stopwatch-size / 4;
+$minute-hand-width: $minute-hand-heigth / 10;
+$second-hand-heigth: $stopwatch-size / 2;
```

Comment:
There is a typo in the variable name $second-hand-heigth. It should be $second-hand-height.